app.js

```diff
@@ -1,10 +1,10 @@
 // Set constraints for the video stream
-var constraints = { video: { facingMode: "environment" }, audio: false };
+const constraints = { video: { facingMode: "environment" }, audio: false };
```

Comment:
no reason to ever have `var` anymore. const = constant, let = letting(a variable that can  be updated)